files/brews/hub.rb

```diff
@@ -0,0 +1,22 @@
+require 'formula'
+
+class Hub < Formula
+  VERSION = "2.2.0-rc1"
```

Comment:
Cool, didn't [`version` is available](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/formula.rb#L176-L178) in the formula. Hope that there's a documentation somewhere about all available property.